examples/build-and-run/README.md

```diff
@@ -8,12 +8,20 @@ for nbody, our example application.
 bueno build -s Dockerfile.mpich -t nbody
 ```
 Once completed, there will be a new tarball in the acting directory. We'll use
+<<<<<<< HEAD
```

Comment:
It looks like you didn't resolve merge conflicts. This is an example of one of them.